cmd/run/run.go

```diff
@@ -410,6 +424,11 @@ func DefaultConfig() *Config {
 			Addr:                "0.0.0.0:2112",
 			EnableRPCHistograms: false,
 		},
+		CheckQueryCache: CheckQueryCache{
+			Enabled: true,
```

Comment:
The default should be `false` IMO. Developers should have to opt-in to eventually consistent semantics in OpenFGA. The default right now is read-committed data, which implies strongly consistent semantics. If a developer want to trade off stale reads for lower latency, then that's their choice, but we shouldn't change the default behavior.